src/styles/App.css

```diff
@@ -0,0 +1,254 @@
+@import url("./library.css");
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+#root {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  overflow: hidden;
+  font-size: 14px;
+}
+
+body {
+  background: #f1f4f8;
+  position: fixed;
+  overflow: hidden;
+  overscroll-behavior-y: none;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  color: black;
+  -webkit-font-smoothing: antialiased;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.home-header {
+  position: absolute;
+  height: 4rem;
+  width: 100%;
+  padding: 1.2em 2.2em;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #111;
```

Comment:
here too

Reply:
Thanks for pointing this out! I made `var` for the colors I'm using.